crates/torii/grpc/src/server/subscriptions/model_diff.rs

```diff
@@ -231,13 +233,15 @@ where
                     }
 
                     Ok(MaybePendingStateUpdate::PendingUpdate(_)) => {
-                        debug!(target = "subscription", "ignoring pending state update {block_num}")
+                        debug!(target = LOG_TARGET, block_number = %block_num, "Ignoring pending state update.")
                     }
 
                     Err(e) => {
                         error!(
-                            target = "subscription",
-                            "failed to fetch state update for block {block_num}: {e}"
+                            target = LOG_TARGET,
+                            block_num = %block_num,
+                            error = %e,
+                            "While fetching state update for block."
```

Comment:
```suggestion
                            "Fetching state update."
```
Two comments here:
1. The `when`, `while` are not necessary I think. We should describe the action.
2. When structured logging is not use, we have those sentences like "Doing X for Y {y}". But when structure logging, we describe the action into the sentence, and the arguments gives the context.

For example:
```
Event emitted keys. keys="0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"
```
Should be
```
Event emitted. keys="0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"
```
Does this make sense to you too?